tripper/triplestore.py

```diff
@@ -18,10 +18,13 @@
 import re
 import warnings
 from collections.abc import Sequence
-from datetime import datetime
 from importlib import import_module
 from typing import TYPE_CHECKING
 
+from .errors import NamespaceError, TriplestoreError, UniquenessError
+from .literal import Literal, en
+from .namespace import DCTERMS, DM, FNO, MAP, OWL, RDF, RDFS, XML, XSD, Namespace
```

Comment:
```suggestion
from tripper.errors import NamespaceError, TriplestoreError, UniquenessError
from tripper.literal import Literal, en
from tripper.namespace import DCTERMS, DM, FNO, MAP, OWL, RDF, RDFS, XML, XSD, Namespace
```
Relative imports should not be used if it can be helped. The way they work has been changed throughout the minor Python 3 versions and it is not as runtime "stable" as absolute imports.
`__init__.py` files are the exception to this rule.